docs/api/target_properties.rst

```diff
@@ -155,13 +155,21 @@ blt_print_target_properties
 
 .. code-block:: cmake
 
-    blt_print_target_properties(TARGET <target>)
+    blt_print_target_properties(TARGET     <target name>
+                                CHILDREN   <TRUE|FALSE>
+                                PROP_REGEX <regular_expression_string>)
 
 Prints out all properties of the given target.
 
 TARGET
   Name of CMake target
 
+CHILDREN
+  Whether or not to print the properties of the target's children (false by default)
+
+PROP_REGEX
+  Limit the properties to print (all by default)
```

Comment:
Before merging, please ensure that the arguments in the docs match those of the actual macro in the code.